src/main/webapp/app/arm/arm.component.ts

```diff
@@ -31,5 +32,16 @@ export class ArmComponent implements OnInit {
             this.armInput.arm_suspended = '';
         }
     }
-
+    checkboxChange(event, checked) {
+        if (checked) {
+            this.armInput.arm_type = event.target.value;
+        } else {
+            this.armInput.arm_type = '';
+        }
+    }
+    displayDrugName(drugs: Array<Drug>) {
```

Comment:
I pointed out in other pr. You could just do Drug[] for the type. Easier for you.

Reply:
Basically most variables use `Array` instead of `[]`. We need to change that in lot of places.